policies/modules/ROOT/pages/policies-included-apply.adoc

```diff
@@ -5,9 +5,9 @@ endif::[]
 :keywords: policy, endpoint
 :page-aliases: api-manager::using-policies.adoc
 
-In Connected Mode and Mule Gateway, you apply included policies via *API Manager*. The following steps demonstrate this.
+In Flex Gateway Connected Mode and Mule Gateway, you apply included policies via *API Manager*. The following steps demonstrate this.
 
-In Local Mode, you apply policies via an `ApiInstance` YAML resource. For information about applying policies in Local Mode, see the xref:gateway::flex-local-configuration-reference-guide.adoc#api-instance[Declarative Configuration Reference Guide].
+In Flex Gateway Local Mode, you apply policies via an `ApiInstance` YAML resource. For information about applying policies in Flex Gateway Local Mode, see the xref:gateway::flex-local-configuration-reference-guide.adoc#api-instance[Declarative Configuration Reference Guide].
```

Comment:
```suggestion
For Flex Gateway running in Local Mode, you apply policies via an `ApiInstance` YAML resource. For information about applying policies in Local Mode, see the xref:gateway::flex-local-configuration-reference-guide.adoc#api-instance[Declarative Configuration Reference Guide].
```

Probably don't need the second "Flex Gateway" since we've already said "Flex Gateway running in Local Mode" in the previous sentence. It seemed redundant.